rpm_docker.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+sudo service docker start
+docker build -t centos:latest .
+docker cp `docker ps -l -q`:/root/rpmbuild/RPMS/x86_64/${PACKAGE}-${VERSION}-1.x86_64.rpm ./
```

Comment:
When `package_cloud` runs on CentOS container, gem requires `LANG UTF-8`.

I set ENV `LANG=en_US.UTF8` but same message appered and failed... :crying_cat_face: 

Do you know any ideas? Please help